src/test/java/org/sagebionetworks/bridge/BridgeUtilsTest.java

```diff
@@ -656,7 +657,7 @@ public void parseIntegerOrDefault() {
     
     @Test(expectedExceptions = BadRequestException.class)
     public void parseIntegerOrDefaultThrowsException() {
-        BridgeUtils.getIntOrDefault("asdf", 3);
+        assertEquals(BridgeUtils.getIntOrDefault("asdf", 3), 3);
```

Comment:
Minor: This assert will never actually check anything, since getIntOrDefault just throws.

Reply:
Removed